sdk/core/azure-core/tests/azure_core_asynctests/test_basic_transport.py

```diff
@@ -11,13 +11,24 @@
 except ImportError:
     import mock
 
-from azure.core.pipeline.transport import HttpRequest, AsyncHttpResponse, AsyncHttpTransport
+from azure.core.pipeline.transport import HttpRequest, AsyncHttpResponse, AsyncHttpTransport, AioHttpTransport
 from azure.core.pipeline.policies import HeadersPolicy
 from azure.core.pipeline import AsyncPipeline
 
 import pytest
 
 
+@pytest.mark.asyncio
+async def test_basic_options_aiohttp():
+
+    request = HttpRequest("OPTIONS", "https://httpbin.org")
```

Comment:
I'm not a big fan of calling arbitrary endpoints in our tests. Any way to avoid this?

Reply:
That's a bigger problem than this PR, so created a dedicated issue:
https://github.com/Azure/azure-sdk-for-python/issues/9324